hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1552,6 +1552,14 @@ public enum OperationStatusCode {
     "hbase.regionserver.slowlog.systable.enabled";
   public static final boolean DEFAULT_SLOW_LOG_SYS_TABLE_ENABLED_KEY = false;
 
+  @Deprecated
```

Comment:
> And why adding a deprecated field here? Just remove it? 

We are replacing `hbase.slowlog.systable.chore.duration` with `hbase.regionserver.named.queue.chore.duration`. Since the old config property already exists, we will have to follow deprecation cycle. 

> Better not adding more things to HConstants, please move then to a more specific location.

Sounds good.